core/frontend/src/components/zenoh-inspector/ZenohInspector.vue

```diff
@@ -0,0 +1,274 @@
+                <template v-for="(item, i) in filtered_topics">
+                  <v-list-item
+                    :key="i"
+                    :value="item"
+                    active-class="deep-purple--text text--accent-4"
+                  >
+                    <template #default="{ active }">
+                      <v-list-item-content>
+                        <v-list-item-title>
+                          {{ item }}
+                        </v-list-item-title>
+                      </v-list-item-content>
+
+                      <v-list-item-action>
+                        <v-radio
+                          :input-value="active"
+                          color="deep-purple accent-4"
+                        />
+                      </v-list-item-action>
+                    </template>
+                  </v-list-item>
+                </template>
+              </v-list-item-group>
+            </v-list>
+          </v-card>
+        </v-sheet>
+      </v-col>
+
+      <v-col
+        sm="8"
+      >
+        <v-card
+          v-if="selected_topic"
+          outlined
+          width="100%"
+          height="700px"
+        >
+          <v-card-title>
+            {{ selected_topic }}
+            <v-chip
+              v-tooltip="'Topic liveliness status'"
+              :color="topic_liveliness[selected_topic] === undefined ? 'grey'
+                : (topic_liveliness[selected_topic] ? 'green' : 'red')"
+              class="ml-2"
+            >
+              {{ topic_liveliness[selected_topic] === undefined ? 'Unknown'
+                : (topic_liveliness[selected_topic] ? 'Alive' : 'Dead') }}
+            </v-chip>
+            <v-chip
+              v-tooltip="'Topic type'"
+              color="blue"
+              class="ml-2"
+            >
+              {{ topic_types[selected_topic] || 'Unknown' }}
+            </v-chip>
+            <v-chip
+              v-tooltip="'Topic message serialization type'"
+              color="purple"
+              class="ml-2"
+            >
+              {{ topic_typs[selected_topic] || 'Unknown' }}
+            </v-chip>
+          </v-card-title>
+          <v-card-text
+            style="overflow: auto; height: calc(100% - 48px);"
+          >
+            <pre>{{ formatMessage(current_message) }}</pre>
+          </v-card-text>
+        </v-card>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+<script lang="ts">
+import {
+  Config, Sample, SampleKind, Session, Subscriber,
+} from '@eclipse-zenoh/zenoh-ts'
+import Vue from 'vue'
+
+interface ZenohMessage {
+  topic: string
+  payload: string
+  timestamp: Date
+}
+
+export default Vue.extend({
+  name: 'ZenohInspector',
+  data() {
+    return {
+      topics: [] as string[],
+      messages: {} as { [key: string]: ZenohMessage },
+      topic_liveliness: {} as { [key: string]: boolean },
+      topic_types: {} as { [key: string]: string },
+      topic_typs: {} as { [key: string]: string },
```

Comment:
I think this name was not intentional. Maybe the `topic_msg_types`would be better.

```suggestion
      topic_msg_types: {} as { [key: string]: string },
```